### DISCUSSION TITLE
Copilot student plan error

### DISCUSSION THREAD

**Discussion by Tomtunn** [community/community#65454] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hello,

I've been using the student plan for GitHub Copilot without any issues until today. However, I encountered the following error message:
```
[INFO] [auth] [2023-08-30T11:32:50.082Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-08-30T11:32:50.083Z] GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as Tomtunn."
```
I also checked my plan status, and it indicates that my plan is not activated, even though my student plan is still valid and has not expired.

Could someone please assist me in resolving this issue?

**Comment by davecheney** [comment, marked as answer]
@Tomtunn your student-2021 coupon was valid for two years and expired on July 31st of this year.

**Reply by Tomtunn** [reply]
I recently resubmitted my activation request, and while it has been approved, I'm still unable to use it. Is there a possible technical issue or step I might have overlooked?
<img width="250" alt="image" src="https://github.com/community/community/assets/88201742/efe1f2de-f0e0-42ff-980b-d7b5e4974be8">

**Reply by davecheney** [reply]
I don't see the coupon on your account yet, there may be 24 hours delay. Once the coupon is applied to your account, when you https://github.com/github-copilot/signup you will be able to choose the free student plan.

**Reply by Tomtunn** [reply]
Thank you for your assistance. I now have access to Copilot, but there seems to have been about a 3-day delay.

**Comment by mdtanvirahamedshanto** [comment]
expired on July 31st of this year.

**Comment by fsantamaria1** [comment]
Do you have multiple emails listed on your account? 
I had the same issue and I was able to fix it by signing in using the .edu email I used to submit the application.

**Reply by InvincibleWyq** [reply]
@fsantamaria1  
Thanks for your reply! I do have multiple e-mails associated with my GitHub account, and I am a verified student.
However using the .edu email to sign in does not work for me

**Reply by InvincibleWyq** [reply]
Thanks, problem solved.
Just wait for 3~4 days. I get my students account approved on Sept.1st and get Copilot on Sept.4th.

**Reply by fsantamaria1** [reply]
I'm glad you could figure it out!

**Comment by InvincibleWyq** [comment]
@davecheney 
Dear developer: 
I am having similar issue. 

My Copilot stop working yesterday, so I re-activate my student pack benefit, which has already been approved. Still, Copilot refuse to work. After pressing 'sign up' button, it directs me to payment page.

<img src="https://github.com/community/community/assets/37479394/8cdb3660-ef10-4759-8a8a-54c3efbb28cd" alt="Image" width="600">
<img src="https://github.com/community/community/assets/37479394/868b5eea-ad6c-4072-94f6-ed94eedb8b97" alt="Image" width="300">
<img src="https://github.com/community/community/assets/37479394/9ac56e93-1ba2-44c5-9128-f3b43cf123aa" alt="Image" width="300">

Much thanks! 
Yiqin

**Reply by InvincibleWyq** [reply]
Thanks, problem solved.
Just wait for 3~4 days. I get my students account approved on Sept.1st and get Copilot on Sept.4th.

**Comment by jvj00** [comment]
I have the same issue explained above: I tried to sign out and into GitHub from VSCode, but nothing happened.
![image](https://github.com/community/community/assets/94923288/c5ac2a69-d188-4d09-897e-31530ee8a990)
Moreover, my academic subscription has not expired, as you can see below.
![image](https://github.com/community/community/assets/94923288/8aca47a0-abb6-4175-853d-94facd225c2f)
![image](https://github.com/community/community/assets/94923288/4d37ba8a-e2b1-4624-ad66-d23cb5370d0c)
And if I try to go to the copilot page, in the settings of GitHub, and then click on "Start free trial", I'll be redirected to the same page where I am.
